### PR TITLE
[TS-1866] read-db converts `byte array` to HEX string instead of prin…

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**
…ting as object hash code

### CODE REVIEW

File: core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt [hunk 1]
```diff
@@ -45,6 +45,7 @@ import com.google.common.util.concurrent.ThreadFactoryBuilder
 import com.google.protobuf.UnsafeByteOperations
 import com.opencsv.CSVWriterBuilder
 import io.netty.buffer.Unpooled
+import io.netty.util.internal.StringUtil.toHexString
```
Comment (OptimumCode) [inline core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt:48]: I believe we should not use Netty's internal stuff. Could you please use `io.netty.buffer.ByteBufUtil.hexDump(byte[])` instead if it matches the requirements?

File: core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt [hunk 2]
```diff
@@ -268,6 +269,7 @@ private fun Any.toStringValue(): String = when (this) {
     is BigDecimal -> stripTrailingZeros().toPlainString()
     is Double -> toBigDecimal().toStringValue()
     is Float -> toBigDecimal().toStringValue()
+    is ByteArray -> toHexString(this)
```
Comment (OptimumCode) [inline core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt:272]: Also, I think we should add a test for that. Using `blob` in MySQL should do the thing

Comment (Nikita-Smirnov-Exactpro) [reply core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt]: Added `data` field with `blob` type into DataBaseReaderIntegrationTest

File: core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt
Comment (OptimumCode) [inline core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt:1]: Why do we interfere with the existing test logic? We need a test that shows that the ByteArray is converted as expected. Right now I don't see that in the test

**Review by OptimumCode — CHANGES_REQUESTED** [review]

File: core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt
Comment (Nikita-Smirnov-Exactpro) [reply core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt:1]: The current test shows, that field with `blob` type and ByteArray java type can be read and written.
As I understand you want to add a separate test for serialization TableRaw to CSV, am I right?

Comment (OptimumCode) [reply core/src/test/kotlin/com/exactpro/th2/read/db/app/DataBaseReaderIntegrationTest.kt:1]: I want to describe the behavior you added in a test so anybody would see that this is how it is expected in the output. That is all I propose

**Review by OptimumCode — APPROVED** [review]

**Pull request merged**